api/Dockerfile

```diff
@@ -1,7 +1,7 @@
-FROM node:15.10.0-buster-slim
+FROM node:18.15.0-buster-slim
 
 ENV DEBIAN_FRONTEND=noninteractive
-
+RUN npm i -g npm@latest
```

Comment:
not required - let the node container use whatever npm version is installed in it.

Reply:
@HexF I fixed it